app/pages/_app.tsx

```diff
@@ -3,6 +3,7 @@ import React from 'react';
 import Head from 'next/head';
 import { IntlWrapper } from 'lib-pintl';
 import { useRouter } from 'next/router';
+import { DeviceProvider } from '../wrappers/device'
```

Comment:
Our components library has `DeviceProvider` implementation. Don't need to create duplication.